### PR TITLE
Show and control the migration of metadata in the task manager

### PR DESCRIPTION
**Pull request by matthias-ronge**
This is solved so that for each project to be migrated a separate thread is created. This was the easiest way to do it and will (hopefully) make the migration even faster.

### CODE REVIEW

**Comment by Kathrin-Huber** [comment]
please write tests!

File: Kitodo/src/main/java/org/kitodo/production/forms/MigrationForm.java [hunk 1]
```diff
@@ -57,25 +59,28 @@
...
     public void migrateMetadata() {
+        migrateWorkflow = false;
         try {
-            migrationService.migrateMetadata();
+            allProjects = ServiceManager.getProjectService().getAll();
+            projectListShown = true;
         } catch (DAOException e) {
-            Helper.setErrorMessage(ERROR_DATABASE_READING, new Object[] {ObjectType.PROCESS.getTranslationSingular() },
-                logger, e);
+            Helper.setErrorMessage("Error during database access");
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/kitodo/production/forms/MigrationForm.java]: Why did you remove the `logger` and `e` parameters from method call? Maybe / helpful information get lost.

Comment (matthias-ronge) [reply Kitodo/src/main/java/org/kitodo/production/forms/MigrationForm.java]: I deleted the whole method and put another method here, which has a catch in the same place, but another helper call. This only looks like the diff view, as if I had just changed the one line. The modification process was different, so I did not notice.

**Comment by matthias-ronge** [comment]
Should everything be fixed now.
